lib/chromic_pdf/pdf/connection/inet.ex

```diff
@@ -58,7 +58,7 @@ if Code.ensure_loaded?(WebSockex) do
       :inets.start()
 
       url = String.to_charlist("http://#{host}:#{port}/json/version")
-      headers = [{'accept', 'application/json'}]
+      headers = [{~c"accept", ~c"application/json"}]
```

Comment:
:thinking: new elixir version makes formatter do funny things

Reply:
resolved by pinning the local elixir version to 1.14 as that's the one the `lint` job uses